demos/react-router/package.json

```diff
@@ -20,11 +26,9 @@
       "last 1 safari version"
     ]
   },
-  "dependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.1",
-    "use-ackee": "^3.0.1"
+  "eslintConfig": {
```

Comment:
This is necessary to prevent CRA from looking up the directory tree and finding+complaining about the package's own ESLint config. This sets it back to the default lint CRA rules.